eli5/xgboost.py

```diff
@@ -91,7 +93,9 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        expand_missing_features=False,
```

Comment:
I haven't tried it, so likely you're right :) my point was that if xgboost makes decision based on these features then why shouldn't we show them. Maybe some of them have large weights and showing them is more important than showing features which are not missing. There can be a lot of present features as well; we have "top" and "feature_re" arguments to limit the display.